<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepWindow</class>
 <widget class="QDialog" name="StepWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Step</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>951</width>
     <height>451</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="TimeDiagram">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>941</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="stepWindow">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>560</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Current instruction being executed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="instNameLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
